src/NServiceBus.AcceptanceTests/Recoverability/When_message_is_moved_to_error_queue.cs

```diff
@@ -54,6 +54,30 @@ public Task May_send_outgoing_messages(TransportTransactionMode transactionMode)
             .Run();
         }
 
+        [TestCase(TransportTransactionMode.SendsAtomicWithReceive)]
+        [TestCase(TransportTransactionMode.TransactionScope)]
+        [TestCase(TransportTransactionMode.ReceiveOnly)]
+        [TestCase(TransportTransactionMode.None)]
```

Comment:
not too happy about this. Seems it would be good enough to run this test with the transports default tx settings instead. But that would require a entire new endpoint config only for logging. Opinions?